paris-processor/src/main/java/com/airbnb/paris/processor/ParisProcessor.kt

```diff
@@ -106,26 +107,29 @@ class ParisProcessor : SkyProcessor() {
 
         rFinder.processStyleables(styleablesInfo)
 
-        val styleablesTree = StyleablesTree(styleablesInfo + externalStyleablesInfo)
+        val allStyleables = styleablesInfo + externalStyleablesInfo
+        val styleablesTree = StyleablesTree(allStyleables)
         for (styleableInfo in styleablesInfo) {
             StyleApplierJavaClass(styleablesTree, styleableInfo).write()
         }
 
-        if (!styleablesInfo.isEmpty()) {
+        if (styleablesInfo.isNotEmpty()) {
             try {
                 ModuleJavaClass(styleablesInfo).write()
             } catch (e: ProcessorException) {
                 Errors.log(e)
             }
         }
 
-        if (!styleablesInfo.isEmpty() || !externalStyleablesInfo.isEmpty()) {
+        if (allStyleables.isNotEmpty()) {
             try {
-                check(rFinder.element != null) {
+
+                checkNotNull(rFinder.element) {
                     "Unable to locate R class. Please annotate an arbitrary package with @ParisConfig and set the rClass parameter to the R class."
                 }
 
                 val parisClassPackageName = rFinder.element!!.packageName
+                KotlinStyleExtensionsFile("$parisClassPackageName.paris.extensions", allStyleables).write()
```

Comment:
Not sure if there is a better package to use

Reply:
LGTM.